common/src/main/java/com/lowdragmc/lowdraglib/gui/widget/DraggableScrollableWidgetGroup.java

```diff
@@ -391,17 +392,40 @@ else if (yBarWidth > 0 && isOnYScrollPane(mouseX, mouseY)) {
                 setFocus(true);
                 return true;
             }
+            for (int i = widgets.size() - 1; i >= 0; i--) {
+                Widget widget = widgets.get(i);
+                if (widget.isVisible() && widget.isMouseOverElement(mouseX, mouseY)) {
+                    return widget.mouseClicked(mouseX, mouseY, button);
+                }
+            }
```

Comment:
mouseClicked of the children widgets has alreadly been called in the `checkClickedDragged`?